src/app/ui/earn/EarnPageSkeleton.tsx

```diff
@@ -1,3 +1,15 @@
+'use client';
```

Comment:
why use client?

Reply:
Only because if any of this children uses `styled` components they can't be rendered on the server-side